data/json/items/chemicals_and_resources.json

```
@@ -1462,5 +1462,69 @@
     "explode_in_fire": true,
     "explosion": { "power": 20, "shrapnel": 0 },
     "flags": [ "NO_DROP", "IRREMOVABLE" ]
+  },
+  {
+    "type": "AMMO",
+    "id": "chem_dnp",
+    "category": "chems",
+    "price": 5000,
+    "name": "DNP",
+    "name_plural": "DNP",
+    "symbol": "=",
+    "color": "yellow",
+    "description": "2,4-Dinitrophenol, a yellow crystaline powder giving off distinct chemical smell.  It seems to have a strange propensity to stain everything in its vincinity yellow.  Mostly useful for its explosive qualities.",
+    "material": "powder",
+    "//": "1.68g/ccm, WWI era high-explosive, TNT-RE 0.81, metabolic poison/dieting drug, readily sublimates",
```

```suggestion
    "//": "1.68g/ccm, WWI era high-explosive, commercial pesticide, TNT-RE 0.81, metabolic poison, readily sublimates",
```

Given that this is an abusable thing, can we not put 'dieting' drug here, even with the comments?